python/sdk/merlin/docker/pyfunc.Dockerfile

```diff
@@ -12,18 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG BASE_IMAGE
-ARG MODEL_PATH
-
+ARG BASE_IMAGE=ghcr.io/caraml-dev/merlin/merlin-pyfunc-base:0.38.1
 FROM ${BASE_IMAGE}
 
-COPY ${MODEL_PATH} /model
+# Download and install user model dependencies
+ARG MODEL_DEPENDENCIES_URL
+COPY ${MODEL_DEPENDENCIES_URL} conda.yaml
+RUN conda env create --name merlin-model --file conda.yaml
+
+# Copy and install pyfunc-server and merlin-sdk dependencies
+COPY merlin/python/pyfunc-server /pyfunc-server
+COPY merlin/python/sdk /sdk
+ENV SDK_PATH=/sdk
 
-# Disable pip caching to minimize image size
-ENV PIP_NO_CACHE_DIR=1
```

Comment:
Do we not need this anymore? 😮

Reply:
We might still need it. However, I removed this to make local Dockerfile as close as possible to primary Dockerfile used to build image remotely.